templates/pvc.yaml

```diff
@@ -31,13 +34,28 @@ kind: PersistentVolumeClaim
 metadata:
   name: dispatcher-filelogdir
 spec:
-  storageClassName: {{ .Values.storage_class_rwmany }} 
+  storageClassName: {{ .Values.storage_class }} 
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
+      #- ReadWriteMany
   resources:
     requests:
       storage: 30Gi
+# PersistentVolumeClaims related to dda and ddcache are useless in the dispatcher if dda is not deployed
+
+{{- if .Values.mount_dda }}
 ---
-# PersistentVolumeClaims related to dda and ddcache should be defined only in dda, since they are useless in the dispatcher if dda is not deployed
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-reduced-ddcache
+spec:
+  storageClassName: {{ .Values.storage_class }} 
+  accessModes:
+    - ReadWriteOnce      #- ReadWriteMany
+  resources:
+    requests:
+      storage: 30Gi
+{{ end }}
```

Comment:
But if the same PVC is also created in the dda-chart, this duplication will produce an error. 
But you have to set mount_dda=true to actually mount it.
Seems like this situation require two different flags

Reply:
well, since we now always create it here and not in dda, it does not matter. But it can be put a bit more clearly. I will modify the variable names etc.

Reply:
I renamed. DDA will be adopted accordingly.

Reply:
thanks for the comments!